ui/ozone/platform/wayland/wayland_window.cc

```diff
@@ -204,6 +204,8 @@ bool WaylandWindow::HasCapture() {
 }
 
 void WaylandWindow::Show() {
+  set_keyboard_focus(true);
```

Comment:
does that affect tooltips? i.e. do we transfer focus keyboard to tooltips windows?

- for example, place focus on an input, part of chrome or content.
- place mouse cursor over an element with tooltip
(tooltip shows and keyboard focus is set to it).

does it have side effect? when is it keyboard unfocused?